pkg/harvester/l10n/en-us.yaml

```diff
@@ -893,7 +893,7 @@ harvester:
       label: Alertmanager Configs
       diabledTips:
         prefix: 'You must enable'
-        middle: 'Alertmanager'
+        middle: 'Monitoring'
         suffix: 'for configs to take effect.'
       disabledAddon:
         prefix: 'Monitoring Addon is disabled now, click'
```

Comment:
@DaiYuzeng :
It could be a bit complex here:
When monitoring addon is not enabled, hint  like `enable .... addon at first ...`;  when addon is enabled, then hint `enable alertmanager to take effect`

Reply:
Got it, thanks!